src/components/templates/TemplateWorkflowsContent.vue

```diff
@@ -104,6 +100,7 @@ const selectFirstTab = () => {
 watch(isReady, selectFirstTab, { once: true })
 
 const workflowLoading = ref<string | null>(null)
+const currentViewMode = ref<'card' | 'list'>('card')
```

Comment:
Is there any specific reason why view mode state is owned by `TemplateWorkflowView`? If not, let's move it to `TemplateWorkflowView`.